reporters/src/main/java/com/nextdoor/bender/monitoring/ReporterUtils.java

```diff
@@ -0,0 +1,101 @@
+package com.nextdoor.bender.monitoring;
+
+import com.amazonaws.services.cloudwatch.model.Dimension;
+import com.amazonaws.services.cloudwatch.model.StandardUnit;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.nextdoor.bender.monitoring.cw_embedded_metrics.AWSMetricObject;
+import com.nextdoor.bender.monitoring.cw_embedded_metrics.CloudWatchMetricObject;
+import com.nextdoor.bender.monitoring.cw_embedded_metrics.MetricPOJO;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ * Utils class to capture static methods used across multiple reporters
+ */
+public class ReporterUtils {
+    public static List<Dimension> tagsToDimensions(final Set<Tag> tags) {
+        return tags.stream().map(e -> tagToDim(e.getKey(), e.getValue())).collect(Collectors.toList());
+    }
+
+    private static Dimension tagToDim(String name, String value) {
+        return new Dimension().withName(name).withValue(value != null ? value : "None");
+    }
+
+    /**
+     * Example of accepted format:
+     * {
+     *   "_aws": {
+     *     "Timestamp": 1574109732004,
+     *     "CloudWatchMetrics": [
+     *       {
+     *         "Namespace": "lambda-function-metrics",
+     *         "Dimensions": [["functionVersion"]],
+     *         "Metrics": [
+     *           {
+     *             "Name": "time",
+     *             "Unit": "Milliseconds"
+     *           }
+     *         ]
+     *       }
+     *     ]
+     *   },
+     *   "functionVersion": "$LATEST",
+     *   "time": 100,
+     *   "requestId": "989ffbf8-9ace-4817-a57c-e4dd734019ee"
+     * }
+     *
+     * Java code example: https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Embedded_Metric_Format_Generation_PutLogEvents.html
+     */
+    public static String getCloudWatchEmbeddedMetricsJson(String namespace,
```

Comment:
This method should be moved into the embedded metrics class path since it's not generic enough for others to use.

Reply:
I'll have to leave it as public static if I move it there in order to test the serialization method.